hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -4006,19 +4009,23 @@ TermIndex installCheckpoint(String leaderId, DBCheckpoint omDBCheckpoint)
       throws Exception {
 
     Path checkpointLocation = omDBCheckpoint.getCheckpointLocation();
+    Path checkpointDataDir = Paths.get(checkpointLocation.getParent().toString(),
+        OM_CHECKPOINT_DATA_DIR);
+    Path omDbPath = Paths.get(checkpointDataDir.toString(), OM_DB_NAME);
     TransactionInfo checkpointTrxnInfo = OzoneManagerRatisUtils
-        .getTrxnInfoFromCheckpoint(configuration, checkpointLocation);
+        .getTrxnInfoFromCheckpoint(configuration, omDbPath);
 
     LOG.info("Installing checkpoint with OMTransactionInfo {}",
         checkpointTrxnInfo);
 
-    return installCheckpoint(leaderId, checkpointLocation, checkpointTrxnInfo);
+    return installCheckpoint(leaderId, checkpointDataDir, checkpointTrxnInfo);
   }
 
-  TermIndex installCheckpoint(String leaderId, Path checkpointLocation,
+  TermIndex installCheckpoint(String leaderId, Path checkpointDataDir,
```

Comment:
checkstyle: installCheckpoint() method too long. Opened https://issues.apache.org/jira/browse/HDDS-13895 to refactor the checkpointing code.